test/kubemark/stop-kubemark.sh

```diff
@@ -35,9 +35,9 @@ RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
 
 detect-project &> /dev/null
 
-"${KUBECTL}" delete -f "${RESOURCE_DIRECTORY}/addons" --namespace="kubemark" &> /dev/null || true
-"${KUBECTL}" delete -f "${RESOURCE_DIRECTORY}/hollow-node.yaml" --namespace="kubemark" &> /dev/null || true
-"${KUBECTL}" delete -f "${RESOURCE_DIRECTORY}/kubemark-ns.json" &> /dev/null || true
+timeout 300 "${KUBECTL}" delete -f "${RESOURCE_DIRECTORY}/addons" --namespace="kubemark" &> /dev/null || true
+timeout 300 "${KUBECTL}" delete -f "${RESOURCE_DIRECTORY}/hollow-node.yaml" --namespace="kubemark" &> /dev/null || true
+timeout 300 "${KUBECTL}" delete -f "${RESOURCE_DIRECTORY}/kubemark-ns.json" &> /dev/null || true
```

Comment:
IIUC kubectl delete already supports timeout param, right?

Why not kubectl delete --timeout=X?

Reply:
Fixed